torchvision/prototype/features/_image.py

```diff
@@ -16,7 +16,22 @@
 class ColorSpace(StrEnum):
     OTHER = StrEnum.auto()
     GRAYSCALE = StrEnum.auto()
+    GRAYSCALE_ALPHA = StrEnum.auto()
     RGB = StrEnum.auto()
+    RGBA = StrEnum.auto()
```

Comment:
This naming is bothering me a bit. I was thinking that maybe `RGB_ALPHA` makes sense because it aligns with `GRAYSCALE_ALPHA`. On the other hand `RGBA` is a thing in PIL. So I'm on the fence, thoughts?